english/hit-song/hit-song.md

```diff
@@ -0,0 +1,7 @@
+You write a hit song called "No one wants to Pika-choose me".  But unfortunately your agent steals all of the royalties to your song.
+
+What will you do next?
+
+[Begin to dance to pass the time?](dance/dance.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.